### PR TITLE
adding configurable timeout to ldap connections

### PR DESCRIPTION
**Pull request by idob-cloudify**

### CODE REVIEW

File: cfy_manager/components/restservice/config/cloudify-rest.conf [hunk 1]
```diff
@@ -12,6 +12,7 @@
     ldap_domain: '{{ restservice.ldap.domain }}',
     ldap_is_active_directory: {{ restservice.ldap.is_active_directory }},
     ldap_dn_extra: '{{ restservice.ldap.dn_extra }}',
+    ldap_timeout: 30.0,
```
Comment (isaac-s) [inline cfy_manager/components/restservice/config/cloudify-rest.conf]: We need a `config.yaml` entry for this.

Comment (yarivgiga) [reply cfy_manager/components/restservice/config/cloudify-rest.conf]: i think he added one

Comment (isaac-s) [reply cfy_manager/components/restservice/config/cloudify-rest.conf]: @yarivgiga nope

Comment (yarivgiga) [reply cfy_manager/components/restservice/config/cloudify-rest.conf]: yeah i see now, i confused it with config.py

**Pull request merged**